### PR TITLE
[ES] Native review of `B` articles

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

Native review of B articles listed in the `No native review` section according to osu! wiki status in spanish

### CODE REVIEW

File: wiki/Brand_identity_guidelines/es.md
Comment (agatemosu) [inline wiki/Brand_identity_guidelines/es.md:1]: puedes cambiar "logotipo" por "logo"

File: wiki/Beatmapping/Combo/es.md [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
 
-**El Combo** (o **comboset**) se refiere a un conjunto de "[Hit Circles](/wiki/Gameplay/Hit_object/Hit_circle)" y "[Sliders](/wiki/Gameplay/Hit_object/Slider)" en [osu!](/wiki/Game_mode/osu!) y [osu!catch](/wiki/Game_mode/osu!catch) [beatmaps](/wiki/Beatmap). [Hit objects](/wiki/Gameplay/Hit_object) del mismo combo, comparten un [combo colour](/wiki/Beatmapping/Combo_colour) y están conectados por lineas de seguimiento. El final del combo, otorgará un bonus de [health](/wiki/Gameplay/Health) si se completa correctamente. Cada combo comienza con un "Hit Object" marcado con un nuevo color.
+**El combo** (o **comboset**) se refiere a un conjunto de «[círculos](/wiki/Gameplay/Hit_object/Hit_circle)» y «[sliders](/wiki/Gameplay/Hit_object/Slider)» en [beatmaps](/wiki/Beatmap) de [osu!](/wiki/Game_mode/osu!) y [osu!catch](/wiki/Game_mode/osu!catch). Los [objetos](/wiki/Gameplay/Hit_object) del mismo combo, comparten un [color](/wiki/Beatmapping/Combo_colour) y están conectados por líneas de seguimiento. El final del combo, otorgará un bonus de [salud](/wiki/Gameplay/Health) si se completa correctamente. Cada combo comienza con un «objeto» marcado con un nuevo color.
```
Comment (agatemosu) [inline wiki/Beatmapping/Combo/es.md:16]:
```suggestion
El **combo** (o **comboset**) se refiere a un conjunto de «[círculos](/wiki/Gameplay/Hit_object/Hit_circle)» y «[sliders](/wiki/Gameplay/Hit_object/Slider)» en [beatmaps](/wiki/Beatmap) de [osu!](/wiki/Game_mode/osu!) y [osu!catch](/wiki/Game_mode/osu!catch). Los [objetos](/wiki/Gameplay/Hit_object) del mismo combo, comparten un [color](/wiki/Beatmapping/Combo_colour) y están conectados por líneas de seguimiento. Al final del combo, otorgará un bonus de [salud](/wiki/Gameplay/Health) si se completa correctamente. Cada combo comienza con un «objeto» marcado con un nuevo color.
```

File: wiki/Beatmapping/Combo/es.md [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
 
 [osu!taiko](/wiki/Game_mode/osu!taiko) y [osu!mania](/wiki/Game_mode/osu!mania) no usan el sistema de combos.
 
 ## osu!
 
-En [osu!](/wiki/Game_mode/osu!), cada objeto dentro de un combo tiene su propio numero de combo, ademas de compartir el mismo color. Los combos nuevos comienzan en 1 y cuentan hacia arriba, mientras que los "[Spinners](/wiki/Gameplay/Hit_object/Spinner)" fuerzan al siguiente objeto a iniciar un nuevo combo. Mas vida es recuperada con las puntuaciones [Geki](/wiki/Gameplay/Judgement/Geki) and [Katu](/wiki/Gameplay/Judgement/Katu), que se otorgan al impactar correctamente el ultimo objeto de un combo.
+En [osu!](/wiki/Game_mode/osu!), cada objeto dentro de un combo tiene su propio número de combo, además de compartir el mismo color. Los combos nuevos comienzan en 1 y cuentan hacia arriba, mientras que los «[spinners](/wiki/Gameplay/Hit_object/Spinner)» fuerzan al siguiente objeto a iniciar un nuevo combo. Más salud es recuperada con las puntuaciones [Geki](/wiki/Gameplay/Judgement/Geki) y [Katu](/wiki/Gameplay/Judgement/Katu), que se otorgan al impactar correctamente el último objeto de un combo.
```
Comment (agatemosu) [inline wiki/Beatmapping/Combo/es.md:22]:
```suggestion
En [osu!](/wiki/Game_mode/osu!), cada objeto dentro de un combo tiene su propio número de combo, además de compartir el mismo color. Los combos nuevos comienzan en 1 y cuentan hacia arriba, mientras que los «[spinners](/wiki/Gameplay/Hit_object/Spinner)» fuerzan al siguiente objeto a iniciar un nuevo combo. Más salud es recuperada con los resultados [Geki](/wiki/Gameplay/Judgement/Geki) y [Katu](/wiki/Gameplay/Judgement/Katu), que se pueden obtener en el último objeto de un combo.
```

File: wiki/Beatmapping/Combo/es.md [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
 
 ## osu!catch
 
-En [osu!catch](/wiki/Game_mode/osu!catch), cada tipo de [fruta](/wiki/Gameplay/Hit_object/Fruit) está coloreada en el orden indicado. Los "[Juices Streams](/wiki/Gameplay/Hit_object/Juice_stream)" y las frutas al final de los "Sliders" **siempre** tendrán el mismo color que la fruta inicial, sin embargo, las [bananas](/wiki/Gameplay/Hit_object/Banana) (o platanos) siempre tendrán su respectivo color amarilo. Cuando un combo es completado, las frutas en el plato del jugador explotarán hacia afuera.
+En [osu!catch](/wiki/Game_mode/osu!catch), cada tipo de [fruta](/wiki/Gameplay/Hit_object/Fruit) está coloreada en el orden indicado. Los «[juices streams](/wiki/Gameplay/Hit_object/Juice_stream)» y las frutas al final de los «sliders» **siempre** tendrán el mismo color que la fruta inicial, sin embargo, las [bananas](/wiki/Gameplay/Hit_object/Banana) (o plátanos) siempre tendrán su respectivo color amarilo. Cuando un combo es completado, las frutas en el plato del jugador explotarán hacia afuera.
```
Comment (agatemosu) [inline wiki/Beatmapping/Combo/es.md:26]:
```suggestion
En [osu!catch](/wiki/Game_mode/osu!catch), cada tipo de [fruta](/wiki/Gameplay/Hit_object/Fruit) está coloreada en el orden indicado. Las «[corrientes de jugo](/wiki/Gameplay/Hit_object/Juice_stream)» y las frutas al final de los «sliders» **siempre** tendrán el mismo color que la fruta inicial, sin embargo, los [plátanos](/wiki/Gameplay/Hit_object/Banana) siempre tendrán su respectivo color amarilo. Cuando un combo es completado, las frutas en el plato del jugador explotarán hacia afuera.
```

Comment (agatemosu) [reply wiki/Beatmapping/Combo/es.md:26]: lo siento

File: wiki/Beatmap/Guest_difficulty/es.md [hunk 4]
```diff
@@ -1,28 +1,30 @@
...
 
-En la [página de discusión del mapa](/wiki/Beatmap_discussion), el anfitrión del mapa puede transferir la propiedad de las dificultades de invitados a sus respectivos mapeadores a través del botón "Propietario de la dificultad". Esto permite que los propietarios de las dificultades de invitados resuelvan los problemas pendientes en sus dificultades por su cuenta.
+En la [página de discusión del beatmap](/wiki/Beatmap_discussion), el anfitrión del beatmap puede transferir la propiedad de las dificultades de invitados a sus respectivos mappers a través del botón «Dueño de la dificultad». Esto permite que los propietarios de las dificultades de invitados resuelvan los problemas pendientes en sus dificultades por su cuenta.
 
-La propiedad de una dificultad de invitado no se puede cambiar si el mapa de ritmos está [Calificado](/wiki/Beatmap/Category#calificados), [Clasificado](/wiki/Beatmap/Category#clasificados), o [Amado](/wiki/Beatmap/Category#amados).
+La propiedad de una dificultad de invitado no se puede cambiar si el beatmap está [calificado](/wiki/Beatmap/Category#calificados), [clasificado](/wiki/Beatmap/Category#clasificados), o [amado](/wiki/Beatmap/Category#amados).
```
Comment (agatemosu) [inline wiki/Beatmap/Guest_difficulty/es.md:30]:
```suggestion
La propiedad de una dificultad de invitado no se puede cambiar si el beatmap está [calificado](/wiki/Beatmap/Category#calificados), [clasificado](/wiki/Beatmap/Category#clasificados) o [amado](/wiki/Beatmap/Category#amados).
```

File: wiki/Beatmap_Spotlights/Seasons/2020_Summer/es.md [hunk 5]
```diff
@@ -37,8 +36,8 @@ A partir de ahora, unirse a las salas multijugador semanales requiere que los pa
 2. Cada sala multijugador temporal proporciona una lista de juego predefinida de beatmaps.
    - Cada lista de juego es un subconjunto de la lista completa de beatmaps seleccionados para la temporada actual.
    - Cada lista de juego contiene una distribución uniforme de beatmaps Hard, Insane y Expert.
-   - Cada lista de juego se corresponde con la letra asignada a cada semana. Por lo tanto, cada lista de juego se reproduce 3 veces durante una temporada.
-3. Dentro de una sala temporal, todos compiten por una tabla de clasificación específica de la sala.
+   - Cada lista de juego se corresponde con la letra asignada a cada semana. Por lo tanto, cada lista de juego se vuelve a jugar 3 veces durante una temporada.
```
Comment (agatemosu) [inline wiki/Beatmap_Spotlights/Seasons/2020_Summer/es.md:39]:
```suggestion
   - Cada lista de juego se corresponde con la letra asignada a cada semana. Por lo tanto, cada lista de juego se puede jugar 3 veces durante una temporada.
```

File: wiki/Beatmap_Spotlights/Seasons/2021_Spring/es.md [hunk 6]
```diff
@@ -36,8 +35,8 @@ A partir de ahora, unirse a las salas multijugador semanales requiere que los pa
 2. Cada sala multijugador temporal proporciona una lista de juego predefinida de beatmaps.
    - Cada lista de juego es un subconjunto de la lista completa de beatmaps seleccionados para la temporada actual.
    - Cada lista de juego contiene una distribución uniforme de beatmaps Hard, Insane y Expert.
-   - Cada lista de juego se corresponde con la letra asignada a cada semana. Por lo tanto, cada lista de juego se reproduce 3 veces durante una temporada.
-3. Dentro de una sala temporal, todos compiten por una tabla de clasificación específica de la sala.
+   - Cada lista de juego se corresponde con la letra asignada a cada semana. Por lo tanto, cada lista de juego se vuelve a jugar 3 veces durante una temporada.
```
Comment (agatemosu) [inline wiki/Beatmap_Spotlights/Seasons/2021_Spring/es.md:38]:
```suggestion
   - Cada lista de juego se corresponde con la letra asignada a cada semana. Por lo tanto, cada lista de juego se puede jugar 3 veces durante una temporada.
```

File: wiki/Beatmap_Spotlights/Seasons/2021_Winter/es.md [hunk 7]
```diff
@@ -36,8 +35,8 @@ A partir de ahora, unirse a las salas multijugador semanales requiere que los pa
 2. Cada sala multijugador temporal proporciona una lista de juego predefinida de beatmaps.
    - Cada lista de juego es un subconjunto de la lista completa de beatmaps seleccionados para la temporada actual.
    - Cada lista de juego contiene una distribución uniforme de beatmaps Hard, Insane y Expert.
-   - Cada lista de juego se corresponde con la letra asignada a cada semana. Por lo tanto, cada lista de juego se reproduce 3 veces durante una temporada.
-3. Dentro de una sala temporal, todos compiten por una tabla de clasificación específica de la sala.
+   - Cada lista de juego se corresponde con la letra asignada a cada semana. Por lo tanto, cada lista de juego se vuelve a jugar 3 veces durante una temporada.
```
Comment (agatemosu) [inline wiki/Beatmap_Spotlights/Seasons/2021_Winter/es.md:38]:
```suggestion
   - Cada lista de juego se corresponde con la letra asignada a cada semana. Por lo tanto, cada lista de juego se puede jugar 3 veces durante una temporada.
```

File: wiki/Beatmap_Spotlights/Seasons/2022_Spring/es.md [hunk 8]
```diff
@@ -32,16 +31,16 @@ A partir de ahora, unirse a las salas multijugador semanales requiere que los pa
    - Después de la primera ola de salas, las listas de juego se abrirán nuevamente semanalmente.
 2. Cada sala de lista de juego proporciona una lista de juego predefinida de beatmaps.
    - Cada lista de juego contiene una distribución uniforme de beatmaps Hard, Insane y Expert.
-   - Cada lista de juego se corresponde con la letra asignada a cada 2 semanas. Por lo tanto, cada lista de juego se reproduce una vez durante una temporada.
-3. Dentro de una sala de lista de juego, todos compiten hacia una tabla de clasificación específica de la sala.
+   - Cada lista de juego se corresponde con la letra asignada a cada 2 semanas. Por lo tanto, cada lista de juego se vuelve a jugar una vez durante una temporada.
```
Comment (agatemosu) [inline wiki/Beatmap_Spotlights/Seasons/2022_Spring/es.md:34]:
```suggestion
   - Cada lista de juego se corresponde con la letra asignada a cada 2 semanas. Por lo tanto, cada lista de juego se repite una vez durante una temporada.
```

File: wiki/Beatmap_Spotlights/Seasons/2022_Summer/es.md [hunk 9]
```diff
@@ -32,16 +31,16 @@ A partir de ahora, unirse a las salas multijugador semanales requiere que los pa
    - Después de la primera ola de salas, las listas de juego se abrirán nuevamente semanalmente.
 2. Cada sala de lista de juego proporciona una lista de juego predeterminada de beatmaps.
    - Cada lista de juego contiene una distribución uniforme de beatmaps Hard, Insane y Expert.
-   - Cada lista de juego se corresponde con la letra asignada a cada 2 semanas. Por lo tanto, cada lista de juego se reproduce una vez durante una temporada.
-3. Dentro de una sala de lista de juego, todos compiten hacia una tabla de clasificación específica de la sala.
+   - Cada lista de juego se corresponde con la letra asignada a cada 2 semanas. Por lo tanto, cada lista de juego se vuelve a jugar una vez durante una temporada.
```
Comment (agatemosu) [inline wiki/Beatmap_Spotlights/Seasons/2022_Summer/es.md:34]:
```suggestion
   - Cada lista de juego se corresponde con la letra asignada a cada 2 semanas. Por lo tanto, cada lista de juego se repite una vez durante una temporada.
```

File: wiki/Beatmap_Spotlights/Seasons/2022_Winter/es.md [hunk 10]
```diff
@@ -32,30 +31,30 @@ A partir de ahora, unirse a las salas multijugador semanales requiere que los pa
    - Después de la primera ola de salas, las listas de juego se abrirán nuevamente semanalmente.
 2. Cada sala de lista de juego proporciona una lista de juego predefinida de beatmaps.
    - Cada lista de juego contiene una distribución uniforme de beatmaps Hard, Insane y Expert.
-   - Cada lista de juego se corresponde con la letra asignada a cada 2 semanas. Por lo tanto, cada lista de juego se reproduce una vez durante una temporada.
-3. Dentro de una sala de lista de juego, todos compiten hacia una tabla de clasificación específica de la sala.
+   - Cada lista de juego se corresponde con la letra asignada a cada 2 semanas. Por lo tanto, cada lista de juego se vuelve a jugar una vez durante una temporada.
```
Comment (agatemosu) [inline wiki/Beatmap_Spotlights/Seasons/2022_Winter/es.md:34]:
```suggestion
   - Cada lista de juego se corresponde con la letra asignada a cada 2 semanas. Por lo tanto, cada lista de juego se repite una vez durante una temporada.
```

File: wiki/Beatmap_Spotlights/Seasons/2023_Winter/es.md [hunk 11]
```diff
@@ -32,16 +31,16 @@ A partir de ahora, unirse a las salas multijugador semanales requiere que los pa
    - Después de la primera ola de salas, las listas de juego se abrirán nuevamente semanalmente.
 2. Cada sala de lista de juego proporciona una lista de juego predeterminada de beatmaps.
    - Cada lista de juego contiene una distribución uniforme de beatmaps Hard, Insane y Expert.
-   - Cada lista de juego se corresponde con la letra asignada a cada 2 semanas. Por lo tanto, cada lista de juego se reproduce una vez durante una temporada.
-3. Dentro de una sala de lista de juego, todos compiten hacia una tabla de clasificación específica de la sala.
+   - Cada lista de juego se corresponde con la letra asignada a cada 2 semanas. Por lo tanto, cada lista de juego se vuelve a jugar una vez durante una temporada.
```
Comment (agatemosu) [inline wiki/Beatmap_Spotlights/Seasons/2023_Winter/es.md:34]:
```suggestion
   - Cada lista de juego se corresponde con la letra asignada a cada 2 semanas. Por lo tanto, cada lista de juego se repite una vez durante una temporada.
```

File: wiki/Beatmap_discussion/es.md [hunk 12]
```diff
@@ -3,51 +3,51 @@ tags:
   - beatmap discussions
   - modding V2
   - MV2
+  - discusiones de beatmaps
 outdated: true
-no_native_review: true
 ---
 
-# Discusión del mapa
+# Discusión del beatmap
```
Comment (agatemosu) [inline wiki/Beatmap_discussion/es.md:10]:
```suggestion
# Discusión de beatmaps
```

File: wiki/Beatmap_discussion/es.md [hunk 13]
```diff
@@ -3,51 +3,51 @@ tags:
...
+La **discusión del beatmap** (también conocida como *Modding v2*) es un sistema para administrar y simplificar el proceso de [modding](/wiki/Modding). Su objetivo es proporcionar una interfaz clara y comprensible, donde se enfoca la mejora de un beatmap. Cuando un beatmap es [enviado](/wiki/Beatmapping/Beatmap_submission), se crea automáticamente una página de discusión junto con la página de información del beatmap. Cuando se actualice el beatmap, la página de discusión también se actualizará. Se puede acceder a la página de discusión haciendo clic en el botón `Discusión` en la página de información. La página de discusión del beatmap consta de algunos elementos (de arriba a abajo):
```
Comment (agatemosu) [inline wiki/Beatmap_discussion/es.md:14]:
```suggestion
La **discusión de beatmaps** (también conocida como *Modding v2*) es un sistema para administrar y simplificar el proceso de [modding](/wiki/Modding). Su objetivo es proporcionar una interfaz clara y comprensible, donde se enfoca la mejora de un beatmap. Cuando un beatmap es [enviado](/wiki/Beatmapping/Beatmap_submission), se crea automáticamente una página de discusión junto con la página de información del beatmap. Cuando se actualice el beatmap, la página de discusión también se actualizará. Se puede acceder a la página de discusión haciendo clic en el botón `Discusión` en la página de información. La página de discusión del beatmap consta de algunos elementos (de arriba a abajo):
```

File: wiki/Beatmap_discussion/es.md (hunk 12 above)
Comment (agatemosu) [inline wiki/Beatmap_discussion/es.md:10]:
```suggestion
# Discusión de beatmaps
```

File: wiki/Beatmap_discussion/es.md (hunk 13 above)
Comment (agatemosu) [inline wiki/Beatmap_discussion/es.md:14]:
```suggestion
La **discusión de beatmaps** (también conocida como *Modding v2*) es un sistema para administrar y simplificar el proceso de [modding](/wiki/Modding). Su objetivo es proporcionar una interfaz clara y comprensible, donde se enfoca la mejora de un beatmap. Cuando un beatmap es [enviado](/wiki/Beatmapping/Beatmap_submission), se crea automáticamente una página de discusión junto con la página de información del beatmap. Cuando se actualice el beatmap, la página de discusión también se actualizará. Se puede acceder a la página de discusión haciendo clic en el botón `Discusión` en la página de información. La página de discusión del beatmap consta de algunos elementos (de arriba a abajo):
```

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**